libs/dh/market-participant/actors/feature-actors/src/lib/dh-actors-overview.component.ts

```diff
@@ -102,9 +118,25 @@ export class DhActorsOverviewComponent implements OnInit, OnDestroy {
         this.loading = false;
       },
     });
+
+    this.dataSource.filterPredicate = dhActorsCustomFilterPredicate();
+
+    const filtersCombined$: Observable<AllFiltersCombined> = combineLatest([
+      this.filters$,
```

Comment:
question: have you considered moving the filtering to GraphQL? or is the data set small enough that we can always fetch every row?

Reply:
Yes, we did. The data set will be around 200 records, so "it shouldn't" be a problem to filter on the client. If we notice perf issues, we'll reconsider :)